Arm/rover_arm_system.hpp

```diff
@@ -3,58 +3,199 @@
+                    Accelerometer::Acceleration_t shoulder_acceleration)
+  {
+    double gravity                = 9.81;
+    double rotunda_acceleration_x = static_cast<double>(rotunda_acceleration.x);
+    double rotunda_acceleration_y = static_cast<double>(rotunda_acceleration.y);
+    double shoulder_acceleration_x =
+        static_cast<double>(shoulder_acceleration.x);
+    double shoulder_acceleration_y =
+        static_cast<double>(shoulder_acceleration.y);
+    if (rotunda_acceleration_x == 0.0)
+    {  // catches division by 0
+      rotunda_acceleration_x = .0001;
+    }
+    if (rotunda_acceleration_y == 0.0)
+    {  // catches division by 0
+      rotunda_acceleration_y = .0001;
+    }
+    if (shoulder_acceleration_x == 0.0)
+    {  // catches division by 0
+      shoulder_acceleration_x = .0001;
+    }
+    if (shoulder_acceleration_y == 0.0)
+    {  // catches division by 0
+      shoulder_acceleration_y = .0001;
+    }
+    double rotunda_angle =
+        acos(gravity / rotunda_acceleration_y) *  // for angle
+        (rotunda_acceleration_x /
+         fabs(rotunda_acceleration_x));  // for direction
+    double shoulder_angle =
+        acos(gravity / shoulder_acceleration_y) *  // for angle
+        (shoulder_acceleration_x /
+         fabs(shoulder_acceleration_x));  // for direction
+    units::angle::degree_t home =
+        static_cast<units::angle::degree_t>(rotunda_angle + shoulder_angle);
+    rotunda_.SetPosition(home);
+  }
+  void HomeElbow(Accelerometer::Acceleration_t rotunda_acceleration,
+                 Accelerometer::Acceleration_t elbow_acceleration)
+  {
+    double gravity                = 9.81;
+    double rotunda_acceleration_x = static_cast<double>(rotunda_acceleration.x);
+    double rotunda_acceleration_y = static_cast<double>(rotunda_acceleration.y);
+    double elbow_acceleration_x   = static_cast<double>(elbow_acceleration.x);
+    double elbow_acceleration_y   = static_cast<double>(elbow_acceleration.y);
+    if (rotunda_acceleration_x == 0.0)
+    {  // catches division by 0
+      rotunda_acceleration_x = .0001;
+    }
+    if (rotunda_acceleration_y == 0.0)
+    {  // catches division by 0
+      rotunda_acceleration_y = .0001;
+    }
+    if (elbow_acceleration_x == 0.0)
+    {  // catches division by 0
+      elbow_acceleration_x = .0001;
+    }
+    if (elbow_acceleration_y == 0.0)
+    {  // catches division by 0
+      elbow_acceleration_y = .0001;
+    }
+    double rotunda_angle =
+        acos(gravity / rotunda_acceleration_y) *  // for angle
+        (rotunda_acceleration_x /
+         fabs(rotunda_acceleration_x));  // for direction
+    double elbow_angle =
+        acos(gravity / elbow_acceleration_y) *  // for angle
+        (elbow_acceleration_x / fabs(elbow_acceleration_x)) *
+        (elbow_acceleration_y / fabs(elbow_acceleration_y));  // for direction
+    units::angle::degree_t home =
+        static_cast<units::angle::degree_t>(rotunda_angle + elbow_angle);
```

Comment:
Same as above - We should build out functions for each of these calculation that way we can more easily test the logic and also cleanup code base.